src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -24,16 +24,27 @@ export const buildCpsAssetPageATIParams = (
   const { metadata, promo } = pageData;
 
   const getChapter1 = pageIdentifier => {
+    if (service === 'news') {
+      return path(['atiAnalytics', 'chapter'], metadata);
+    }
     const chapter = pageIdentifier.split('.')[1];
-    if (['media_asset'].includes(chapter)) {
+    if (['media_asset', 'story'].includes(chapter)) {
       return null;
     }
     return chapter;
   };
 
+  const getProducer = defaultProducer => {
+    if (service === 'news') {
+      return path(['atiAnalytics', 'producerId'], metadata);
+    }
+    return defaultProducer;
+  };
+
```

Comment:
Context: In /news there is some complicated logic around the Chapters and Producers.  However fortunately that logic already exists in Ares - so we just need to take the relevant values from the Ares response for /news assets.